apps/staging-community/package.json

```diff
@@ -12,6 +12,7 @@
     "@grouparoo/bigquery": "^0.2.3-alpha.2",
     "@grouparoo/core": "^0.2.3-alpha.2",
     "@grouparoo/csv": "^0.2.3-alpha.2",
+    "@grouparoo/customerio": "^0.2.3-alpha.2",
```

Comment:
It will be annoying to update all the version numbers this with upstream, so we can wait until the end.